Fixes to README (#139)
direct-framework/digital-research-competencies-framework: README.md

> # DIgital Research CompeTencies (DIRECT) Framework

Reviewer: I guess "Research" should be "REsearch" to indicate the acronym better?

## About

This project aims to define a skills and competencies framework to help us classify and describe technical and non-technical skills we use in our different roles as researchers, RSEs, data specialists, RSE group leads, PIs, etc., along with lists tools/methods/behaviours to demonstrate skills and training materials that can help us gain a particular skill.

A [sister project](https://github.com/direct-framework/direct-webapp) works on implementing a Django webapp to enable practical use of the framework - to browse the skills and competencies, self-assess and create individual skill profiles as “competency wheels”, compare profiles across a team, define template skills for different digital roles (e.g. a data archivist, a data scientists or an RSE with HPC specialism) and other use cases.

## History

> This repository was originally created from the work that happened at the Software Sustainability Institute's Collaborations Workshop 2023 Hack Day.

Reviewer 2: ```suggestion
This repository was originally created from the work that happened at the Software Sustainability Institute's [Collaborations Workshop 2023](https://www.software.ac.uk/workshop/collaborations-workshop-2023-cw23-0) Hack Day.
```

> The idea was to construct a resource on technical skills that we use that is for the RSE community and curated by the RSE community, along with training materials that can help people gain a particular skill, and visualise people's individaul skill profiles as "competency wheels".

Reviewer 2: ```suggestion
The idea was to construct a resource on technical skills that is for the RSE community and curated by the RSE community, along with training materials that can help people gain a particular skill, and visualise people's individual skill profiles as "competency wheels".
```

## Acknowledgements

The initial version of this repository was created during the Software Sustainability Institute Collaborations Workshop

> 2023 Hack Day. Subsequent development was guided by a number of unconference sessions and contirbutions by RSE community members during RSECon23, RSECon23 and CW25.

Reviewer 2: ```suggestion
2023 Hack Day. Subsequent development was guided by a number of unconference sessions and contributions by RSE community members during RSECon23, RSECon23 and CW25.
```
